Linux official build fails

```
[ 98%] Linking CXX executable DDNet
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `free_stream':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:4404: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `parse_packet':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:1464: undefined reference to `av_parser_parse2'
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:1544: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `ff_read_frame_flush':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:1936: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `read_frame_internal':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:1609: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `update_stream_avctx':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:509: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `read_frame_internal':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:1652: undefined reference to `av_parser_init'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `update_stream_avctx':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:509: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `avformat_find_stream_info':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:3679: undefined reference to `av_parser_init'
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:4022: undefined reference to `avcodec_pix_fmt_to_codec_tag'
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:4023: undefined reference to `avpriv_get_raw_pix_fmt_tags'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `estimate_timings_from_pts':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:2825: undefined reference to `av_parser_close'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(utils.o): In function `av_apply_bitstream_filters':
/x86-64/ffmpeg-4.3.1/libavformat/utils.c:5625: undefined reference to `av_bitstream_filter_filter'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(movenc.o): In function `mov_get_rawvideo_codec_tag':
/x86-64/ffmpeg-4.3.1/libavformat/movenc.c:1576: undefined reference to `avpriv_pix_fmt_bps_mov'
/x86-64/ffmpeg-4.3.1/libavformat/movenc.c:1576: undefined reference to `avpriv_pix_fmt_bps_mov'
/x86-64/ffmpeg-4.3.1/libavformat/movenc.c:1576: undefined reference to `avpriv_pix_fmt_bps_mov'
/x86-64/ffmpeg-4.3.1/libavformat/movenc.c:1576: undefined reference to `avpriv_pix_fmt_bps_mov'
/x86-64/ffmpeg-4.3.1/libavformat/movenc.c:1576: undefined reference to `avpriv_pix_fmt_bps_mov'
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(movenc.o):/x86-64/ffmpeg-4.3.1/libavformat/movenc.c:1576: more undefined references to `avpriv_pix_fmt_bps_mov' follow
ddnet-libs/ffmpeg/linux/lib64/libavformat.a(isom.o): In function `ff_mp4_read_dec_config_descr':
/x86-64/ffmpeg-4.3.1/libavformat/isom.c:560: undefined reference to `avpriv_mpa_freq_tab'
collect2: error: ld returned 1 exit status
CMakeFiles/DDNet.dir/build.make:2383: recipe for target 'DDNet' failed
make[3]: *** [DDNet] Error 1
CMakeFiles/Makefile2:301: recipe for target 'CMakeFiles/DDNet.dir/all' failed
make[2]: *** [CMakeFiles/DDNet.dir/all] Error 2
CMakeFiles/Makefile2:268: recipe for target 'CMakeFiles/package_default.dir/rule' failed
make[1]: *** [CMakeFiles/package_default.dir/rule] Error 2
Makefile:251: recipe for target 'package_default' failed
make: *** [package_default] Error 2
```
This is weird since I haven't changed the ffmpeg libs

--- a/cmake/FindFFMPEG.cmake
+++ b/cmake/FindFFMPEG.cmake
@@ -122,8 +122,8 @@ mark_as_advanced(
 )
 
 set(FFMPEG_LIBRARIES
+  ${AVFORMAT_LIBRARY} # has to come before avcodec
   ${AVCODEC_LIBRARY}
-  ${AVFORMAT_LIBRARY}
   ${AVUTIL_LIBRARY}
   ${SWSCALE_LIBRARY}
   ${SWRESAMPLE_LIBRARY}

